ydb/public/lib/ydb_cli/commands/interactive/yql_highlight.h

```diff
@@ -0,0 +1,68 @@
+#pragma once
+
+#include <contrib/restricted/patched/replxx/include/replxx.hxx>
+
+#include <contrib/libs/antlr4_cpp_runtime/src/Token.h>
+#include <contrib/libs/antlr4_cpp_runtime/src/BufferedTokenStream.h>
+#include <contrib/libs/antlr4_cpp_runtime/src/ANTLRInputStream.h>
+
+#include <ydb/public/lib/ydb_cli/commands/interactive/antlr/YQLLexer.h>
+
+#include <regex>
+
+namespace NYdb {
+    namespace NConsoleClient {
+
+        class YQLHighlight final {
+        public:
+            using Color = replxx::Replxx::Color;
+            using Colors = replxx::Replxx::colors_t;
+
+            struct ColorSchema {
+                Color keyword;
+                Color operation;
+                struct {
+                    Color function;
+                    Color type;
+                    Color variable;
+                    Color quoted;
+                } identifier;
+                Color string;
+                Color number;
+                Color unknown;
+
+                static ColorSchema Monaco();
+                static ColorSchema Debug();
```

Comment:
This color schema has better contrast and contains unique colors, so it is applicable for unit and manual testing.